src/core/lv_obj_style.c

```diff
@@ -330,6 +330,7 @@ void lv_obj_set_local_style_prop(lv_obj_t * obj, lv_style_prop_t prop, lv_style_
                                  lv_style_selector_t selector)
 {
     lv_style_t * style = get_local_style(obj, selector);
+    lv_obj_invalidate(obj);
```

Comment:
```suggestion
    if(part == LV_PART_MAIN && lv_style_prop_has_flag(prop, LV_STYLE_PROP_TRANSFORM); lv_obj_invalidate(obj);
```

`LV_STYLE_PROP_TRANSFORM` needs to be added in `lv_style.c/h`.